mopidy/core/actor.py

```diff
@@ -254,10 +254,8 @@ def name(b):
                 has_playlists = b.has_playlists().get()
             except Exception:
                 self.remove(b)
-                logger.exception(
-                    "Fetching backend info for %s failed",
-                    b.actor_ref.actor_class.__name__,
-                )
+                logger.exception("Fetching backend info for %s failed", name(b))
```

Comment:
```suggestion
                logger.exception(f"Fetching backend info for {name(b)} failed")
```

Reply:
Is there a general desire to move logging calls to using f-strings?

Reply:
Yes, we did most (all?) the extensions during the Python 3 port but there was enough to do in Mopidy core as it was, so it's more of a do-it-as-you-go approach here.

Reply:
Sure thing